pyscriptjs/tests/integration/test_zz_examples.py

```diff
@@ -256,7 +256,7 @@ def test_panel_stream(self):
         assert self.page.title() == "PyScript/Panel Streaming Demo"
         wait_for_render(self.page, "*", "<div.*?class=['\"]bk-root['\"].*?>")
 
-    @pytest.mark.skip("flaky test, see issue #864")
+    # @pytest.mark.skip("flaky test, see issue #864")
```

Comment:
This test isn't flaky anymore? Should we remove this line? 😄

Reply:
Yeah, I am supposed to test this today.
Obrigada ✨